tests/sharing/scalability/base_tests.rb

```diff
@@ -0,0 +1,84 @@
+require_relative '../boot'
+
+$stdout.sync = true
```

Comment:
By curiosity, why do you use this?

Reply:
This is needed if we wanted to progressively show the progression of the tasks, as I originally wanted to. So for now, it is not useful